helical/models/caduceus/pretrained_config.py

```diff
@@ -0,0 +1,54 @@
+"""Caduceus config for Hugging Face.
+
+Taken from Caduceus repository: https://github.com/kuleshov-group/caduceus/blob/main/caduceus/configuration_caduceus.py
+"""
+
+from typing import Optional, Union
+
+from transformers import PretrainedConfig
+
+
+class CaduceusPretrainedConfig(PretrainedConfig):
+    """Config that extends the original MambaConfig with params relevant to bi-directionality and RC equivariance."""
+    def __init__(
+            self,
+            # From original MambaConfig
+            d_model: int = 2560,
```

Comment:
currently, for the user, there is no way to set these parameters. Can you make them available in the hydra config?